week-5-practise/js/main.js

```diff
@@ -0,0 +1,64 @@
+
+
+var blueButton = document.querySelector("#blueBtn");
+blueButton.addEventListener("click", changeColorBlueBttn);
+var orangeBttn = document.querySelector("#orangeBtn");
+orangeBttn.addEventListener("click", changeColorOrangeBttn);
+var greenBttn = document.querySelector("#greenBtn");
+greenBttn.addEventListener("click", changeColorGreenBttn)
+function changeColorBlueBttn(){
+    document.querySelector(".jumbotron").style.backgroundColor = "#588fbd";
+    document.querySelector(".buttons .btn-primary").style.backgroundColor = "#ffa500";
+    document.querySelector(".btn-secondary").style.backgroundColor = "black";
+    document.querySelector(".btn-secondary").style.color = "white";
+}
+function changeColorOrangeBttn(){
+    document.querySelector(".jumbotron").style.backgroundColor = "#f0ad4e";
+    document.querySelector(".buttons .btn-primary").style.backgroundColor = `#5751fd`;
```

Comment:
There's quite a lot of duplication here. Do think this is a bad thing? What could we do to improve it?